.github/workflows/ci.yml

```diff
@@ -73,6 +73,30 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Install helm
+        uses: azure/setup-helm@v4
+
+      - name: Update helm chart
+        run: |
+          git checkout main
+          sed -Ei \
+           -e 's/^(version\:) .*/\1 '${{ env.VERSION }}'/g' \
+           -e 's/^(appVersion\:) .*/\1 "'${{ env.VERSION }}'"/g' \
+           charts/dragonfly-operator/Chart.yaml          
+          
+          git commit \
```

Comment:
nit: Is there a need to commit the version into repository? considering we publish the chart as an artifact? :thinking:

Reply:
It's also being done [here](https://github.com/dragonflydb/dragonfly/blob/6b8a8b6410c84f0d03444a6700f54fb806270253/.github/workflows/docker-release.yml#L93), so consistency?